### DISCUSSION TITLE
error : tar (some text) Cannot open: No such file or directory tar: tools: Cannot mkdir: No space left on device

### DISCUSSION THREAD

**Discussion by pazza2234** [skycocker/chromebrew#8315] [Q&A]
I don't got enough space to fully install, if I type ```df``` / directory is 88% full before install or after uninstall. I have Tryed all of the things mentioned in https://github.com/chromebrew/chromebrew/issues/2550 (I got murk mod and fake murk but those things are not that big right?)

**Comment by satmandu** [comment]
> I don't got enough space to fully install, if I type ```df``` / directory is 88% full before install or after uninstall. I have Tryed all of the things mentioned in https://github.com/chromebrew/chromebrew/issues/2550 (I got murk mod and fake murk but those things are not that big right?)

How much space does your device have in total?

**Reply by pazza2234** [reply]
![image](https://github.com/chromebrew/chromebrew/assets/81707002/1978204b-2c98-4f9a-910a-75bae441b6cc)

**Reply by satmandu** [reply]
Can you do a `du -ah /usr/local` ?

**Reply by pazza2234** [reply]
so sorry for long wait @satmandu 
![image](https://github.com/chromebrew/chromebrew/assets/81707002/9144db58-88f7-4f95-a3ec-c994d28d1024)

**Reply by supechicken** [reply]
Your `/usr/local` is not mounted correctly, it should be bind-mounted to `/mnt/stateful_partition/dev_image` by default. (are you installing in guest mode?)

Quick fix:
```shell
sudo mount -B /mnt/stateful_partition/dev_image /usr/local
```

**Reply by pazza2234** [reply]
![image](https://github.com/chromebrew/chromebrew/assets/81707002/0e6fac50-566d-4880-9a32-2e34fd174de0)
also I, earlyer as #2550 told me to do, 
![image](https://github.com/chromebrew/chromebrew/assets/81707002/4fd527da-0f94-4c65-8af1-96e735c4f02a)

**Reply by supechicken** [reply]
Try this:
```shell
sudo mkdir -p /mnt/stateful_partition/dev_image
sudo mount -B /mnt/stateful_partition/dev_image /usr/local
```

**Comment by YeesterPlus** [comment]
do one of 2 things
- just remove stuff
- factory reset via recovery
